src/app/reducers/knowledge.ts

```diff
@@ -1,54 +1,51 @@
-import { compose } from '@ngrx/core';
-import { combineReducers } from '@ngrx/store';
-import { ActionReducer, Action } from '@ngrx/store';
-import { createSelector } from 'reselect';
 import * as knowledge from '../actions/knowledge';
 
-export const CONTENT_CHANGE = 'CONTENT_CHANGE';
-export const IMAGE_CHANGE = 'IMAGE_CHANGE';
-
 export interface State {
   content_response: any,
   image_response: any,
   description_response: any
 }
 
-const initialState: State = {
```

Comment:
I don't think that changing the variable name here was required , both `defaultState` and `inititalState` mean the same .

Reply:
Actually, it happens to be the convention in redux to I applied it here too. @rajvaibhavdubey  Do the other changes look okay?